simtools/db_handler.py

```diff
@@ -975,3 +1045,54 @@ def _getTaggedVersion(self, dbName, version='Current'):
         tags = collection.find(query).sort('_id', pymongo.DESCENDING)[0]
 
         return tags['Tags'][version]['Value']
+
+    def insertFileToDB(self, file, dbName=DB_CTA_SIMULATION_MODEL, **kwargs):
```

Comment:
I think we should add a description of kwargs too

Reply:
Added (linking to the official MongoDB documentation)